support-workers/src/main/scala/com/gu/support/workers/lambdas/Handler.scala

```diff
@@ -12,6 +13,28 @@ import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.util.control.NonFatal
 
+/** this appender makes sure that multi line messages stay together in cloudwatch
+  */
+class LambdaAppender[E] extends OutputStreamAppender[E] {
+  override def start(): Unit = {
+    println("Starting appender " + this.getClass.getName)
```

Comment:
Do we want this?

Reply:
I did that on purpose so that we know that it's working, but I could take it out.  I notice logback already logs quite a lot on startup so maybe we don't want to add to it?  or maybe take it out if it annoys us later